File: XOutput.Api/Websocket/Ds4/Ds4FeedbackResponse.cs
namespace XOutput.Websocket.Ds4
{
    public class Ds4FeedbackResponse : MessageBase
    {
        public const string MessageType = "Ds4Feedback";

        public Ds4FeedbackResponse()
        {
            Type = MessageType;
        }

        public double SmallForceFeedback { get; set; }
        public double BigForceFeedback { get; set; }
    }
}

File: @xoutput/api/src/websocket/ds4/Ds4FeedbackResponse.ts
import { MessageBase } from '../MessageBase';

export const Ds4FeedbackResponseType = 'Ds4Feedback';

export interface Ds4FeedbackResponse extends MessageBase<typeof Ds4FeedbackResponseType> {
  smallForceFeedback: number;
  bigForceFeedback: number;
}
